index.html

```diff
@@ -14,46 +14,258 @@
       rel="stylesheet"
     />
 
-    <title>portfolio-setup Project</title>
+    <title>portfolio-setup</title>
   </head>
   <body>
-    <!-- my-logo section -->
-    <header class="toolbar-section">
-      <div class="logo">
-        <a href="">My Logo</a>
-      </div>
-
-      <div>
-        <p><span class="iconify" data-icon="eva:menu-outline"></span></p>
-      </div>
-    </header>
-
-    <!-- main section -->
-    <main class="main-section">
-      <div class="content">
-        <p class="author">
-          I’m Ismail <br />
-          Glad to see <br />
-          you!
+    <section class="headers-section">
+      <!-- toolbar section -->
+      <section class="toolbar-section">
+        <div>
+          <a href="" class="logo">My Logo </a>
+        </div>
+
+        <p class="menu-logo">
+          <span class="iconify" data-icon="eva:menu-outline"></span>
         </p>
-        <p class="article">
-          I’m a software developer! I can help you build a product , feature or
-          website Look through some of my work and experience! If you like what
-          you see and have a project you need coded, don’t hestiate to contact
-          me.
+      </section>
+
+      <!-- headline section -->
+      <section class="headline-section">
+        <div class="content">
+          <h1 class="author">
+            I’m Ismail <br />
+            Glad to see <br />
+            you!
+          </h1>
+          <p class="article">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+        </div>
+        <div class="let-section">
+          <p class="connect">LET’S CONNECT</p>
+          <img src="img/Ellipse 12.png" alt="Eclipse-image" />
+        </div>
+
+        <div class="eclipse-13">
+          <img src="img/Ellipse 13.png" alt="Eclipse-image" />
+        </div>
+
+        <ul class="social-icons">
+          <li><span class="iconify" data-icon="iconoir:github"></span></li>
+          <li><span class="iconify" data-icon="eva:linkedin-fill"></span></li>
+          <li><span class="iconify" data-icon="fontisto:angelist"></span></li>
+          <li><span class="iconify" data-icon="ei:sc-twitter"></span></li>
+          <li><span class="iconify" data-icon="bx:bxl-medium"></span></li>
+        </ul>
+
+        <div class="eclipse-4">
+          <img src="img/Ellipse 4.png" alt="Eclipse-image" />
+          <img src="img/Rectangle 5.png" alt="Eclipse-image" />
+        </div>
+      </section>
+    </section>
+
+    <!-------------------------------------- middle-section ---------------------------------------->
+
+    <section class="middle-section">
+      <!-- tonic section 1 -->
+      <section class="tonic-section">
+        <div class="tonic-image"></div>
+        <h2 class="tonic">Tonic</h2>
```

Comment:
- [ ] Even thougth you are using the same style here as in the template, your font import is missing the required font weigth. Thats why it looks different. Please add this on top of your css:
`@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');`
This way you will have access to all of Poppins weigths and styles.

## Original 
![image](https://user-images.githubusercontent.com/28903404/143720167-64055a32-5910-43f5-9e28-ef499ed063a0.png)

## Yours
![image](https://user-images.githubusercontent.com/28903404/143720178-5e44cc56-31f0-46d1-966f-5fe925f216ff.png)

## Yours after correct font import
![image](https://user-images.githubusercontent.com/28903404/143720205-e6508e2e-f3d7-4ce4-9998-efcef64b4181.png)